posthog/temporal/batch_exports/bigquery_batch_export.py

```diff
@@ -709,7 +709,7 @@ async def insert_into_bigquery_activity(inputs: BigQueryInsertInputs) -> Records
         record_batch_schema = await wait_for_schema_or_producer(queue, producer_task)
         if record_batch_schema is None:
             external_logger.info(
-                "Batch export finished as there is no data in range %s - %s matching specified filters",
+                "Batch export will finish early as there is no data matching specified filters in range %s - %s",
```

Comment:
Minor re-writing here, just letting the `finish_batch_export_run` be the one who calls that we "finished".